guides/personal_mixed_skin.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: guides
+title: Creating your personal mixed skin
+description: "This guide teaches you how to create your own personal mixed skin."
+current: Guides | Mixing a skin
+---
+
+# Creating your personal mixed skin
+
+This guide will teach you how to create your own personal mix skin. You should know what SD and HD images are. If you don't, please read [this part](https://skinship.xyz/tutorial/introduction#hdsd-elements-aspect-ratios-and-resolution) of the skinning tutorial. You will also need to know the file names of assets in a skin. If you don't, please consult the [skinnable files list](https://docs.google.com/spreadsheets/d/1bhnV-CQRMy3Z0npQd9XSoTdkYxz0ew5e648S00qkJZ8/edit#gid=2074725196) for file names and other bits of information.
+
+Please note that more often than not, the different skins' assets will look wildly different from each other. This means that mixing is often best reserved for mixing gameplay elements, such as `cursor.png`, `cursortrail.png` etc.

```

Comment:
example seems unnecessary and are bascially refering to the same thing. Would just drop it and quit after "gameplay elements"